### ISSUE TITLE
Citra Failed to Launch : api-ms-win-crt-runtime-l1-1-0.dll is missing (Win 7 Ultimate x64)

### ISSUE THREAD

**Issue by sfuns** [citra-emu/citra#1996]
Hey,

i know most of you people use win 8 or 10 and thus make all your new stuff with that to but could you make Citra so it runs with win 7 to?
Many people will say install this but first unistall that , then install this ...
Believe me it will not work cause i did already and just waisted time for nothing, found somewhere that the problem can't be fixed since it's a part of win 8 KB updates, so it won't install if you find what's missing.
It seems to be a more common problem these days because win 7 is getting old, but still, i don't want to use win 8 or 10 & i think many other people thinks the same.
For me they screwed both win 8 & 10 & i'm not planning on using it any time soon.
So could you please remake a version that runs on win 7 ?
Could you let me know if your planning on doing this? 
Sorry for my bad English.
Thank you.

**Comment by wwylele** [comment]
Please read DLL related section in [FAQ](https://github.com/citra-emu/citra/wiki/FAQ)

----
This is the bug tracker for developers of Citra. 

Please Use our [Citra Forum](https://discuss.citra-emu.org/) for questions and support instead. Note that you won't get any support if you
 - use any kinds of non-official build, or
 - use pirated copies of the games, 3DS BIOS files or any copyrighted content.

If you really want to report an issue which is not your personal problem but bugs in Citra, please review [guidelines for contributing](https://github.com/citra-emu/citra/blob/master/CONTRIBUTING.md) carefully, and follow the instructions there.

Closed as invalid

**Comment by sfuns** [comment]
the DLL related section does not help, you can't install visual studio completely, it doesn't solve the problem api-ms-win-crt-runtime-l1-1-0.dll is missing.

**Log from visual when first attempted to install :**

[3190:4210][2016-08-09T14:21:29]i001: Burn v3.7.3813.0, Windows v6.1 (Build 7601: Service Pack 1), path: D:\Bureaublad\vc_redist.x64.exe, cmdline: '-burn.unelevated BurnPipe.{EE6F738D-E2B9-4459-8DDE-8C1E2993C4E7} {758E592A-317C-44B4-ABCD-B086BFBDBC6D} 13192'
[3190:4210][2016-08-09T14:21:29]i000: Setting string variable 'WixBundleLog' to value 'C:\Users\ADMINI~1\AppData\Local\Temp\dd_vcredist_amd64_20160809142129.log'
[3190:4210][2016-08-09T14:21:29]i000: Setting string variable 'WixBundleOriginalSource' to value 'D:\Bureaublad\vc_redist.x64.exe'
[3190:4210][2016-08-09T14:21:29]i000: Setting string variable 'WixBundleOriginalSourceFolder' to value 'D:\Bureaublad\'
[3190:4210][2016-08-09T14:21:29]i000: Setting string variable 'WixBundleName' to value 'Microsoft Visual C++ 2015 Redistributable (x64) - 14.0.23026'
[3190:4210][2016-08-09T14:21:29]i100: Detect begin, 10 packages
[3190:4210][2016-08-09T14:21:29]i000: File search: windows_uCRT_DetectKey, did not find path: C:\Windows\system32\api-ms-win-crt-runtime-l1-1-0.dll
[3190:4210][2016-08-09T14:21:29]i000: File search: windows_uCRT_DetectKeyExists, did not find path: C:\Windows\system32\api-ms-win-crt-runtime-l1-1-0.dll
[3190:4210][2016-08-09T14:21:29]i000: Setting numeric variable 'windows_uCRT_DetectKeyExists' to value 0
[3190:4210][2016-08-09T14:21:29]i102: Detected related bundle: {dab68466-3a7d-41a8-a5cf-415e3ff8ef71}, type: Upgrade, scope: PerMachine, version: 14.0.23918.0, operation: Downgrade
[3190:4210][2016-08-09T14:21:29]i108: Detected compatible package: vcRuntimeMinimum_x64, provider: Microsoft.VS.VC_RuntimeMinimumVSU_amd64,v14, installed: {7B50D081-E670-3B43-A460-0E2CDB5CE984}, version: 14.0.23918, chained: {0D3E9E15-DE7A-300B-96F1-B4AF12B96488}
[3190:4210][2016-08-09T14:21:29]i103: Detected related package: {7B50D081-E670-3B43-A460-0E2CDB5CE984}, scope: PerMachine, version: 14.0.23918.0, language: 0 operation: Downgrade
[3190:4210][2016-08-09T14:21:29]i108: Detected compatible package: vcRuntimeAdditional_x64, provider: Microsoft.VS.VC_RuntimeAdditionalVSU_amd64,v14, installed: {DFFEB619-5455-3697-B145-243D936DB95B}, version: 14.0.23918, chained: {BC958BD2-5DAC-3862-BB1A-C1BE0790438D}
[3190:4210][2016-08-09T14:21:29]i103: Detected related package: {DFFEB619-5455-3697-B145-243D936DB95B}, scope: PerMachine, version: 14.0.23918.0, language: 0 operation: Downgrade
[3190:4210][2016-08-09T14:21:29]i052: Condition '(windows_uCRT_DetectKeyExists AND windows_uCRT_DetectKey >= v10.0.10137.0)' evaluates to false.
[3190:4210][2016-08-09T14:21:29]i052: Condition '(windows_uCRT_DetectKeyExists AND windows_uCRT_DetectKey >= v10.0.10137.0)' evaluates to false.
[3190:4210][2016-08-09T14:21:29]i052: Condition '(windows_uCRT_DetectKeyExists AND windows_uCRT_DetectKey >= v10.0.10137.0)' evaluates to false.
[3190:4210][2016-08-09T14:21:29]i052: Condition '(windows_uCRT_DetectKeyExists AND windows_uCRT_DetectKey >= v10.0.10137.0)' evaluates to false.
[3190:4210][2016-08-09T14:21:29]i052: Condition '(windows_uCRT_DetectKeyExists AND windows_uCRT_DetectKey >= v10.0.10137.0)' evaluates to false.
[3190:4210][2016-08-09T14:21:29]i052: Condition '(windows_uCRT_DetectKeyExists AND windows_uCRT_DetectKey >= v10.0.10137.0)' evaluates to false.
[3190:4210][2016-08-09T14:21:29]i052: Condition '(windows_uCRT_DetectKeyExists AND windows_uCRT_DetectKey >= v10.0.10137.0)' evaluates to false.
[3190:4210][2016-08-09T14:21:29]i052: Condition '(windows_uCRT_DetectKeyExists AND windows_uCRT_DetectKey >= v10.0.10137.0)' evaluates to false.
[3190:4210][2016-08-09T14:21:29]i101: Detected package: vcRuntimeMinimum_x64, state: Obsolete, cached: None
[3190:4210][2016-08-09T14:21:29]i101: Detected package: vcRuntimeAdditional_x64, state: Obsolete, cached: None
[3190:4210][2016-08-09T14:21:29]i101: Detected package: Windows81_x86, state: Absent, cached: None
[3190:4210][2016-08-09T14:21:29]i101: Detected package: Windows81_x64, state: Absent, cached: None
[3190:4210][2016-08-09T14:21:29]i101: Detected package: Windows8_x86, state: Absent, cached: None
[3190:4210][2016-08-09T14:21:29]i101: Detected package: Windows8_x64, state: Absent, cached: None
[3190:4210][2016-08-09T14:21:29]i101: Detected package: Windows7_MSU_x86, state: Absent, cached: None
[3190:4210][2016-08-09T14:21:29]i101: Detected package: Windows7_MSU_x64, state: Absent, cached: None
[3190:4210][2016-08-09T14:21:29]i101: Detected package: WindowsVista_MSU_x86, state: Absent, cached: None
[3190:4210][2016-08-09T14:21:29]i101: Detected package: WindowsVista_MSU_x64, state: Absent, cached: None
[3190:4210][2016-08-09T14:21:29]i052: Condition 'VersionNT64 >= v6.0 OR (VersionNT64 = v5.2 AND ServicePackLevel >= 1)' evaluates to true.
[3190:4210][2016-08-09T14:21:29]i199: Detect complete, result: 0x0
[3190:56F8][2016-08-09T14:21:43]e000: Error 0x80070666: Cannot install a product when a newer version is installed.


**When i uninstalled newer versions and tryed to install again, here is the second log :**

[32A0:08F0][2016-08-09T14:29:34]i001: Burn v3.7.3813.0, Windows v6.1 (Build 7601: Service Pack 1), path: D:\Bureaublad\vc_redist.x64.exe, cmdline: '-burn.unelevated BurnPipe.{DDFFE4F3-917C-4C7D-A698-B46DC4133ACD} {D635B25F-C07B-4D48-B091-E0809C4E7F22} 13900'
[32A0:08F0][2016-08-09T14:29:34]i000: Setting string variable 'WixBundleLog' to value 'C:\Users\ADMINI~1\AppData\Local\Temp\dd_vcredist_amd64_20160809142934.log'
[32A0:08F0][2016-08-09T14:29:34]i000: Setting string variable 'WixBundleOriginalSource' to value 'D:\Bureaublad\vc_redist.x64.exe'
[32A0:08F0][2016-08-09T14:29:34]i000: Setting string variable 'WixBundleOriginalSourceFolder' to value 'D:\Bureaublad\'
[32A0:08F0][2016-08-09T14:29:34]i000: Setting string variable 'WixBundleName' to value 'Microsoft Visual C++ 2015 Redistributable (x64) - 14.0.23026'
[32A0:08F0][2016-08-09T14:29:35]i100: Detect begin, 10 packages
[32A0:08F0][2016-08-09T14:29:35]i000: File search: windows_uCRT_DetectKey, did not find path: C:\Windows\system32\api-ms-win-crt-runtime-l1-1-0.dll
[32A0:08F0][2016-08-09T14:29:35]i000: File search: windows_uCRT_DetectKeyExists, did not find path: C:\Windows\system32\api-ms-win-crt-runtime-l1-1-0.dll
[32A0:08F0][2016-08-09T14:29:35]i000: Setting numeric variable 'windows_uCRT_DetectKeyExists' to value 0
[32A0:08F0][2016-08-09T14:29:35]i052: Condition '(windows_uCRT_DetectKeyExists AND windows_uCRT_DetectKey >= v10.0.10137.0)' evaluates to false.
[32A0:08F0][2016-08-09T14:29:35]i052: Condition '(windows_uCRT_DetectKeyExists AND windows_uCRT_DetectKey >= v10.0.10137.0)' evaluates to false.
[32A0:08F0][2016-08-09T14:29:35]i052: Condition '(windows_uCRT_DetectKeyExists AND windows_uCRT_DetectKey >= v10.0.10137.0)' evaluates to false.
[32A0:08F0][2016-08-09T14:29:35]i052: Condition '(windows_uCRT_DetectKeyExists AND windows_uCRT_DetectKey >= v10.0.10137.0)' evaluates to false.
[32A0:08F0][2016-08-09T14:29:35]i052: Condition '(windows_uCRT_DetectKeyExists AND windows_uCRT_DetectKey >= v10.0.10137.0)' evaluates to false.
[32A0:08F0][2016-08-09T14:29:35]i052: Condition '(windows_uCRT_DetectKeyExists AND windows_uCRT_DetectKey >= v10.0.10137.0)' evaluates to false.
[32A0:08F0][2016-08-09T14:29:35]i052: Condition '(windows_uCRT_DetectKeyExists AND windows_uCRT_DetectKey >= v10.0.10137.0)' evaluates to false.
[32A0:08F0][2016-08-09T14:29:35]i052: Condition '(windows_uCRT_DetectKeyExists AND windows_uCRT_DetectKey >= v10.0.10137.0)' evaluates to false.
[32A0:08F0][2016-08-09T14:29:35]i101: Detected package: vcRuntimeMinimum_x64, state: Absent, cached: None
[32A0:08F0][2016-08-09T14:29:35]i101: Detected package: vcRuntimeAdditional_x64, state: Absent, cached: None
[32A0:08F0][2016-08-09T14:29:35]i101: Detected package: Windows81_x86, state: Absent, cached: None
[32A0:08F0][2016-08-09T14:29:35]i101: Detected package: Windows81_x64, state: Absent, cached: None
[32A0:08F0][2016-08-09T14:29:35]i101: Detected package: Windows8_x86, state: Absent, cached: None
[32A0:08F0][2016-08-09T14:29:35]i101: Detected package: Windows8_x64, state: Absent, cached: None
[32A0:08F0][2016-08-09T14:29:35]i101: Detected package: Windows7_MSU_x86, state: Absent, cached: None
[32A0:08F0][2016-08-09T14:29:35]i101: Detected package: Windows7_MSU_x64, state: Absent, cached: None
[32A0:08F0][2016-08-09T14:29:35]i101: Detected package: WindowsVista_MSU_x86, state: Absent, cached: None
[32A0:08F0][2016-08-09T14:29:35]i101: Detected package: WindowsVista_MSU_x64, state: Absent, cached: None
[32A0:08F0][2016-08-09T14:29:35]i052: Condition 'VersionNT64 >= v6.0 OR (VersionNT64 = v5.2 AND ServicePackLevel >= 1)' evaluates to true.
[32A0:08F0][2016-08-09T14:29:35]i199: Detect complete, result: 0x0
[32A0:08F0][2016-08-09T14:29:37]i200: Plan begin, 10 packages, action: Install
[32A0:08F0][2016-08-09T14:29:37]i052: Condition '(VersionNT64)' evaluates to true.
[32A0:08F0][2016-08-09T14:29:37]i000: Setting string variable 'WixBundleRollbackLog_vcRuntimeMinimum_x64' to value 'C:\Users\ADMINI~1\AppData\Local\Temp\dd_vcredist_amd64_20160809142934_000_vcRuntimeMinimum_x64_rollback.log'
[32A0:08F0][2016-08-09T14:29:37]i000: Setting string variable 'WixBundleLog_vcRuntimeMinimum_x64' to value 'C:\Users\ADMINI~1\AppData\Local\Temp\dd_vcredist_amd64_20160809142934_000_vcRuntimeMinimum_x64.log'
[32A0:08F0][2016-08-09T14:29:37]i052: Condition '(VersionNT64)' evaluates to true.
[32A0:08F0][2016-08-09T14:29:37]i000: Setting string variable 'WixBundleRollbackLog_vcRuntimeAdditional_x64' to value 'C:\Users\ADMINI~1\AppData\Local\Temp\dd_vcredist_amd64_20160809142934_001_vcRuntimeAdditional_x64_rollback.log'
[32A0:08F0][2016-08-09T14:29:37]i000: Setting string variable 'WixBundleLog_vcRuntimeAdditional_x64' to value 'C:\Users\ADMINI~1\AppData\Local\Temp\dd_vcredist_amd64_20160809142934_001_vcRuntimeAdditional_x64.log'
[32A0:08F0][2016-08-09T14:29:37]i052: Condition 'VersionNT = v6.3 AND NOT VersionNT64' evaluates to false.
[32A0:08F0][2016-08-09T14:29:37]w321: Skipping dependency registration on package with no dependency providers: Windows81_x86
[32A0:08F0][2016-08-09T14:29:37]i052: Condition 'VersionNT = v6.3 AND VersionNT64' evaluates to false.
[32A0:08F0][2016-08-09T14:29:37]w321: Skipping dependency registration on package with no dependency providers: Windows81_x64
[32A0:08F0][2016-08-09T14:29:37]i052: Condition 'VersionNT = v6.2 AND NOT VersionNT64' evaluates to false.
[32A0:08F0][2016-08-09T14:29:37]w321: Skipping dependency registration on package with no dependency providers: Windows8_x86
[32A0:08F0][2016-08-09T14:29:37]i052: Condition 'VersionNT = v6.2 AND VersionNT64' evaluates to false.
[32A0:08F0][2016-08-09T14:29:37]w321: Skipping dependency registration on package with no dependency providers: Windows8_x64
[32A0:08F0][2016-08-09T14:29:37]i052: Condition 'VersionNT = v6.1 AND NOT VersionNT64' evaluates to false.
[32A0:08F0][2016-08-09T14:29:37]w321: Skipping dependency registration on package with no dependency providers: Windows7_MSU_x86
[32A0:08F0][2016-08-09T14:29:37]i052: Condition 'VersionNT = v6.1 AND VersionNT64' evaluates to true.
[32A0:08F0][2016-08-09T14:29:37]w321: Skipping dependency registration on package with no dependency providers: Windows7_MSU_x64
[32A0:08F0][2016-08-09T14:29:37]i052: Condition 'VersionNT = v6.0 AND NOT VersionNT64' evaluates to false.
[32A0:08F0][2016-08-09T14:29:37]w321: Skipping dependency registration on package with no dependency providers: WindowsVista_MSU_x86
[32A0:08F0][2016-08-09T14:29:37]i052: Condition 'VersionNT = v6.0 AND VersionNT64' evaluates to false.
[32A0:08F0][2016-08-09T14:29:37]w321: Skipping dependency registration on package with no dependency providers: WindowsVista_MSU_x64
[32A0:08F0][2016-08-09T14:29:37]i201: Planned package: vcRuntimeMinimum_x64, state: Absent, default requested: Present, ba requested: Present, execute: Install, rollback: Uninstall, cache: Yes, uncache: No, dependency: Register
[32A0:08F0][2016-08-09T14:29:37]i201: Planned package: vcRuntimeAdditional_x64, state: Absent, default requested: Present, ba requested: Present, execute: Install, rollback: Uninstall, cache: Yes, uncache: No, dependency: Register
[32A0:08F0][2016-08-09T14:29:37]i201: Planned package: Windows81_x86, state: Absent, default requested: Absent, ba requested: Absent, execute: None, rollback: None, cache: No, uncache: No, dependency: None
[32A0:08F0][2016-08-09T14:29:37]i201: Planned package: Windows81_x64, state: Absent, default requested: Absent, ba requested: Absent, execute: None, rollback: None, cache: No, uncache: No, dependency: None
[32A0:08F0][2016-08-09T14:29:37]i201: Planned package: Windows8_x86, state: Absent, default requested: Absent, ba requested: Absent, execute: None, rollback: None, cache: No, uncache: No, dependency: None
[32A0:08F0][2016-08-09T14:29:37]i201: Planned package: Windows8_x64, state: Absent, default requested: Absent, ba requested: Absent, execute: None, rollback: None, cache: No, uncache: No, dependency: None
[32A0:08F0][2016-08-09T14:29:37]i201: Planned package: Windows7_MSU_x86, state: Absent, default requested: Absent, ba requested: Absent, execute: None, rollback: None, cache: No, uncache: No, dependency: None
[32A0:08F0][2016-08-09T14:29:37]i201: Planned package: Windows7_MSU_x64, state: Absent, default requested: Present, ba requested: Present, execute: Install, rollback: Uninstall, cache: Yes, uncache: No, dependency: None
[32A0:08F0][2016-08-09T14:29:37]i201: Planned package: WindowsVista_MSU_x86, state: Absent, default requested: Absent, ba requested: Absent, execute: None, rollback: None, cache: No, uncache: No, dependency: None
[32A0:08F0][2016-08-09T14:29:37]i201: Planned package: WindowsVista_MSU_x64, state: Absent, default requested: Absent, ba requested: Absent, execute: None, rollback: None, cache: No, uncache: No, dependency: None
[32A0:08F0][2016-08-09T14:29:37]i299: Plan complete, result: 0x0
[32A0:08F0][2016-08-09T14:29:37]i300: Apply begin
[364C:0D4C][2016-08-09T14:29:37]i360: Creating a system restore point.
[364C:0D4C][2016-08-09T14:29:43]i361: Created a system restore point.
[364C:0D4C][2016-08-09T14:29:43]i370: Session begin, registration key: SOFTWARE\Microsoft\Windows\CurrentVersion\Uninstall\{e46eca4f-393b-40df-9f49-076faf788d83}, options: 0x7, disable resume: No
[364C:0D4C][2016-08-09T14:29:43]i000: Caching bundle from: 'C:\Users\ADMINI~1\AppData\Local\Temp\{e46eca4f-393b-40df-9f49-076faf788d83}\.be\VC_redist.x64.exe' to: 'C:\ProgramData\Package Cache\{e46eca4f-393b-40df-9f49-076faf788d83}\VC_redist.x64.exe'
[364C:0D4C][2016-08-09T14:29:43]i320: Registering bundle dependency provider: {e46eca4f-393b-40df-9f49-076faf788d83}, version: 14.0.23026.0
[364C:0D4C][2016-08-09T14:29:43]i371: Updating session, registration key: SOFTWARE\Microsoft\Windows\CurrentVersion\Uninstall\{e46eca4f-393b-40df-9f49-076faf788d83}, resume: Active, restart initiated: No, disable resume: No
[364C:6038][2016-08-09T14:29:43]i304: Verified existing payload: vcRuntimeMinimum_x64 at path: C:\ProgramData\Package Cache\.unverified\vcRuntimeMinimum_x64.
[364C:6038][2016-08-09T14:29:43]i305: Verified acquired payload: vcRuntimeMinimum_x64 at path: C:\ProgramData\Package Cache\.unverified\vcRuntimeMinimum_x64, moving to: C:\ProgramData\Package Cache\{0D3E9E15-DE7A-300B-96F1-B4AF12B96488}v14.0.23026\packages\vcRuntimeMinimum_amd64\vc_runtimeMinimum_x64.msi.
[364C:6038][2016-08-09T14:29:43]i304: Verified existing payload: cab5046A8AB272BF37297BB7928664C9503 at path: C:\ProgramData\Package Cache\.unverified\cab5046A8AB272BF37297BB7928664C9503.
[364C:6038][2016-08-09T14:29:43]i305: Verified acquired payload: cab5046A8AB272BF37297BB7928664C9503 at path: C:\ProgramData\Package Cache\.unverified\cab5046A8AB272BF37297BB7928664C9503, moving to: C:\ProgramData\Package Cache\{0D3E9E15-DE7A-300B-96F1-B4AF12B96488}v14.0.23026\packages\vcRuntimeMinimum_amd64\cab1.cab.
[364C:6038][2016-08-09T14:29:43]i304: Verified existing payload: vcRuntimeAdditional_x64 at path: C:\ProgramData\Package Cache\.unverified\vcRuntimeAdditional_x64.
[364C:6038][2016-08-09T14:29:43]i305: Verified acquired payload: vcRuntimeAdditional_x64 at path: C:\ProgramData\Package Cache\.unverified\vcRuntimeAdditional_x64, moving to: C:\ProgramData\Package Cache\{BC958BD2-5DAC-3862-BB1A-C1BE0790438D}v14.0.23026\packages\vcRuntimeAdditional_amd64\vc_runtimeAdditional_x64.msi.
[364C:6038][2016-08-09T14:29:43]i304: Verified existing payload: cab2C04DDC374BD96EB5C8EB8208F2C7C92 at path: C:\ProgramData\Package Cache\.unverified\cab2C04DDC374BD96EB5C8EB8208F2C7C92.
[364C:6038][2016-08-09T14:29:43]i305: Verified acquired payload: cab2C04DDC374BD96EB5C8EB8208F2C7C92 at path: C:\ProgramData\Package Cache\.unverified\cab2C04DDC374BD96EB5C8EB8208F2C7C92, moving to: C:\ProgramData\Package Cache\{BC958BD2-5DAC-3862-BB1A-C1BE0790438D}v14.0.23026\packages\vcRuntimeAdditional_amd64\cab1.cab.
[364C:6038][2016-08-09T14:29:43]i304: Verified existing payload: Windows7_MSU_x64 at path: C:\ProgramData\Package Cache\.unverified\Windows7_MSU_x64.
[364C:6038][2016-08-09T14:29:43]i305: Verified acquired payload: Windows7_MSU_x64 at path: C:\ProgramData\Package Cache\.unverified\Windows7_MSU_x64, moving to: C:\ProgramData\Package Cache\42D5BEC7DDFBD49E76467529CBC2868987BF8460\packages\Patch\x64\Windows6.1-KB2999226-x64.msu.
[364C:0D4C][2016-08-09T14:29:43]i301: Applying execute package: vcRuntimeMinimum_x64, action: Install, path: C:\ProgramData\Package Cache\{0D3E9E15-DE7A-300B-96F1-B4AF12B96488}v14.0.23026\packages\vcRuntimeMinimum_amd64\vc_runtimeMinimum_x64.msi, arguments: ' MSIFASTINSTALL="7" NOVSUI="1"'
[32A0:08F0][2016-08-09T14:29:44]i319: Applied execute package: vcRuntimeMinimum_x64, result: 0x0, restart: None
[364C:0D4C][2016-08-09T14:29:44]i325: Registering dependency: {e46eca4f-393b-40df-9f49-076faf788d83} on package provider: Microsoft.VS.VC_RuntimeMinimumVSU_amd64,v14, package: vcRuntimeMinimum_x64
[364C:0D4C][2016-08-09T14:29:44]i301: Applying execute package: vcRuntimeAdditional_x64, action: Install, path: C:\ProgramData\Package Cache\{BC958BD2-5DAC-3862-BB1A-C1BE0790438D}v14.0.23026\packages\vcRuntimeAdditional_amd64\vc_runtimeAdditional_x64.msi, arguments: ' MSIFASTINSTALL="7" NOVSUI="1"'
[32A0:08F0][2016-08-09T14:29:45]i319: Applied execute package: vcRuntimeAdditional_x64, result: 0x0, restart: None
[364C:0D4C][2016-08-09T14:29:45]i325: Registering dependency: {e46eca4f-393b-40df-9f49-076faf788d83} on package provider: Microsoft.VS.VC_RuntimeAdditionalVSU_amd64,v14, package: vcRuntimeAdditional_x64
[364C:0D4C][2016-08-09T14:29:45]i301: Applying execute package: Windows7_MSU_x64, action: Install, path: C:\ProgramData\Package Cache\42D5BEC7DDFBD49E76467529CBC2868987BF8460\packages\Patch\x64\Windows6.1-KB2999226-x64.msu, arguments: '"C:\Windows\SysNative\wusa.exe" "C:\ProgramData\Package Cache\42D5BEC7DDFBD49E76467529CBC2868987BF8460\packages\Patch\x64\Windows6.1-KB2999226-x64.msu" /quiet /norestart'
[364C:0D4C][2016-08-09T14:29:47]e000: Error 0x80240017: Failed to execute MSU package.
[32A0:08F0][2016-08-09T14:29:47]e000: Error 0x80240017: Failed to configure per-machine MSU package.
[32A0:08F0][2016-08-09T14:29:47]i319: Applied execute package: Windows7_MSU_x64, result: 0x80240017, restart: None
[32A0:08F0][2016-08-09T14:29:47]e000: Error 0x80240017: Failed to execute MSU package.
[364C:0D4C][2016-08-09T14:29:47]i372: Session end, registration key: SOFTWARE\Microsoft\Windows\CurrentVersion\Uninstall\{e46eca4f-393b-40df-9f49-076faf788d83}, resume: ARP, restart: None, disable resume: No
[364C:0D4C][2016-08-09T14:29:47]i371: Updating session, registration key: SOFTWARE\Microsoft\Windows\CurrentVersion\Uninstall\{e46eca4f-393b-40df-9f49-076faf788d83}, resume: ARP, restart initiated: No, disable resume: No
[32A0:08F0][2016-08-09T14:29:47]i399: Apply complete, result: 0x80240017, restart: None, ba requested restart:  No


So in other words, i need to have api-ms-win-crt-runtime-l1-1-0.dll in path: C:\Windows\system32\api-ms-win-crt-runtime-l1-1-0.dll , but how can i get it?

Win 7 Update does not offer support anymore, and i'm guessing you can only find it in win 8 or 10 so if you have it, could you zip it and put it here on an attachement?
Thank you.

**Comment by wwylele** [comment]
@sfuns Please go to [our Forum](https://discuss.citra-emu.org/)
